pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go

```diff
@@ -246,7 +240,6 @@ func (i *KubeAPIServerInformers) GetOpenshiftUserInformers() userinformer.Shared
 }
 
 func (i *KubeAPIServerInformers) Start(stopCh <-chan struct{}) {
-	i.InternalKubernetesInformers.Start(stopCh)
```

Comment:
After this change, what is starting the informers?

Reply:
Why would we care, if we're entirely dropping internal informers we don't need to. External informers are started in the next line, or I'm missing something.

Reply:
internalInformers are still consumed within this file by the authorizer: https://github.com/openshift/origin/blob/master/pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go#L78

Since we are no longer starting internal kube informers in that method, I have updated `patchContext.informerStartFuncs` here https://github.com/openshift/origin/pull/21411/files#diff-01a8de689f656fdb0f202987ee6c1f2cR148